services/api/src/typeDefs.js

```diff
@@ -1416,6 +1416,7 @@ const typeDefs = gql`
     Get an organization by its ID
     """
     organizationById(organization: Int!): Organization
+    organizationByName(organization: String!): Organization
```

Comment:
Might be better to use `name: String!` instead here to reduce confusion.

I'm also seeing that `organizationById` should probably use `id` instead of `organization` but that's a different issue

Reply:
The resolver will need to be updated too

Reply:
Yep makes sense, updated now.